src/wasm.rs

```diff
@@ -38,3 +43,93 @@ fn init_panic_hook() {
     console_error_panic_hook::set_once();
     init_console_log();
 }
+
+// thread_local! {
+//     static KV_STORES: HashMap<String, Box<dyn Store>> = Default::default();
+// }
+
+thread_local! {
+    pub static KV_STORES: RefCell<HashMap<String, Arc<dyn Store>>> = Default::default();
```

Comment:
In the old world the state was kept in an async function with an iloop. Now it is a global and the async iloop is gone. The calls just originate from JS.